cluster/images/hyperkube/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM google/debian:wheezy
```

Comment:
We use backports for our container vm -- would it make sense to use that here as well (so that this is as closer to VMs on GCE)?